lefthook.yml

```diff
@@ -0,0 +1,4 @@
+pre-commit:
+  commands:
+    lint:
+      run: standardrb --fix
```

Comment:
You may need also need `stage_fixed: https://github.com/evilmartians/lefthook/blob/master/docs/configuration.md#stage_fixed